inline.go

```diff
@@ -111,17 +111,17 @@ func emphasis(p *parser, out *bytes.Buffer, data []byte, offset int) int {
 
 func codeSpan(p *parser, out *bytes.Buffer, data []byte, offset int) int {
 	data = data[offset:]
-
+	datalen := len(data)
```

Comment:
"data" and "len" are separate words, right? If so, this should be `dataLen` according to idiomatic Go style, see https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps.